server/routes/admin.js

```diff
@@ -0,0 +1,29 @@
+import { Router } from 'express'
+
+import { collections } from '../utils/mongoCollections'
+import ensureAdmin from '../utils/passport/ensure-admin'
+
+const router = Router()
+
+router.route('/api/v1/admin/:uid').patch(ensureAdmin, async (req, res) => {
```

Comment:
if the `:uid` value is a user's uid then this route should be `/api/v1/admin/users/:uid`. As it stands now it suggests that the `:uid` is an admin's uid.

Reply:
4c885b5249ccbc5570f1e915cc0ac3d7d6147536